packages/firestore/src/protos/update.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+
+# Variables
+PROTOS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+WORK_DIR=`mktemp -d`
+
+# check if tmp dir was created
+if [[ ! "$WORK_DIR" || ! -d "$WORK_DIR" ]]; then
+  echo "Could not create temp dir"
+  exit 1
+fi
+
+# deletes the temp directory on exit
+function cleanup {
+  rm -rf "$WORK_DIR"
+  echo "Deleted temp working directory $WORK_DIR"
+}
+
+# register the cleanup function to be called on the EXIT signal
+trap cleanup EXIT
+
+# Enter work dir
+pushd "$WORK_DIR"
+
+# Clone necessary git repos.
+git clone git@github.com:googleapis/googleapis.git
+git clone git@github.com:google/protobuf.git
+
+# Copy necessary protos.
+mkdir -p "${PROTOS_DIR}/google/api"
+cp googleapis/google/api/{annotations.proto,http.proto} \
+   "${PROTOS_DIR}/google/api/"
+
+mkdir -p "${PROTOS_DIR}/google/firestore/v1beta1"
+cp googleapis/google/firestore/v1beta1/*.proto \
+   "${PROTOS_DIR}/google/firestore/v1beta1/"
+
+mkdir -p "${PROTOS_DIR}/google/rpc"
+cp googleapis/google/rpc/status.proto \
+   "${PROTOS_DIR}/google/rpc/"
+
+mkdir -p "${PROTOS_DIR}/google/type"
+cp googleapis/google/type/latlng.proto \
+   "${PROTOS_DIR}/google/type/"
+
+mkdir -p "${PROTOS_DIR}/google/protobuf"
+cp protobuf/src/google/protobuf/{any.proto,empty.proto,struct.proto,timestamp.proto,wrappers.proto} \
```

Comment:
If you wanted to make this shorter this could factor out .proto too.

```
protobuf/src/google/protobuf/{any,empty,struct,timestamp,wrappers}.proto
```

Reply:
👍